src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -22,6 +22,15 @@
 
     public static final String MESSAGE_INVALID_INDEX = "Index is not a non-zero unsigned integer.";
 
+    /**
+     * Parses {@code String args} and extracts out the type of Person (Tutor/Student).
+     *
+     * @return Returns the string at the index of args.
+     */
+    public static String parsePersonType(String args, int index) {
+        return args.substring(index, index + 1);
+    }
+
```

Comment:
Updated code to not use magic numbers 👍 

Note: if we decide to switch to using enums for t/s, we should make a method for converting "t"/"s" to it's enum counterparts. 

```java
    public static String parsePersonType(String args) {
        // Allows for command to be valid even with multiple whitespaces within the command.
        // For e.g. "add    t   n/..." will be a valid command read as "add t n/...".
        String formattedString = args.replaceAll("\\s{2,}", " ").trim();
        String[] parsedString = formattedString.split(" ");
        return (String) Array.get(parsedString, 1);
    }
```

Reply:
Update: 
Since we're using enums, this is the method for it:
``` java
    public static enum parsePersonType(String args) throws ParseException {
        // Allows for command to be valid even with multiple whitespaces within the command.
        // For e.g. "add    t   n/..." will be a valid command read as "add t n/...".
        String formattedString = args.replaceAll("\\s{2,}", " ").trim();
        String[] parsedString = formattedString.split(" ");
        String personType = (String) Array.get(parsedString, 1);
        try (PersonType.detectPersonType(personType)) {
            return detectPersonType(personType);
        } catch (ParseException e) {
            throw e;
        }
    }
```